webapp/components/file_attachment.jsx

```diff
@@ -120,7 +114,7 @@ class FileAttachment extends React.Component {
             var re3 = new RegExp('\\)', 'g');
             var url = fileUrl.replace(re1, '%20').replace(re2, '%28').replace(re3, '%29');
```

Comment:
Not related to your changes, but this could be done with a call to encodeURIComponent. I think it's just really old code though